src/main/java/com/epam/lab/payments/model/AccountRepository.java

```diff
@@ -11,6 +11,6 @@
  * Created by Александр on 03.11.2017.
  */
 public interface AccountRepository extends Repository<Account, Long> {
-    @Query("SELECT p FROM Person p WHERE LOWER(p.lastName) = LOWER(:lastName)")
-    List<Account> find(@Param("user_id") Integer user_id);
+//    @Query("SELECT p FROM Person p WHERE LOWER(p.lastName) = LOWER(:lastName)")
+//    List<Account> find(@Param("user_id") Integer user_id);
 }
```

Comment:
Commented code in git - bad practice
Why simple Repository? Look at JpaRepository or CRUD repository. It contains several functions, which you need. 
And, i think u need to read little bit more about Spring Data. This query seems very strange. 
You only need to correctly write method name and params. And that's all.
BTW, find by last name? WHAT?
https://www.petrikainulainen.net/programming/spring-framework/spring-data-jpa-tutorial-three-custom-queries-with-query-methods/

Reply:
lastname is my bad for pushing template of query annotation